R2Northstar/NorthstarMods
# Hide remote mods in mods display

Hide remote mods in the in-game mods menu, so they cannot be enabled/disabled by players.

- [x] Depends on https://github.com/R2Northstar/NorthstarLauncher/pull/684

Closes #798.

#### How to test

1. Join lobby;
2. Set `allow_mod_auto_download` to 1 through console (`allow_mod_auto_download 1`);
3. `sv_cheats 1`;
4. Download mod: `script NSDownloadMod("nyami.mp_brick", "1.0.2")`;
5. Reload mods: `reload_mods`;
6. Go back to the main screen and check the mods list.

With this branch, the mod should not appear anymore.

#### Screenshots

###### Without this

![image](https://github.com/R2Northstar/NorthstarMods/assets/11993538/a261d1c3-e36d-45d3-ae62-9cd4f43ef9eb)

###### With this

![image](https://github.com/R2Northstar/NorthstarMods/assets/11993538/36b1ea3a-46bf-418e-b8a7-ce0006e7e030)
